### DISCUSSION TITLE
LTE dongle not working after update

### DISCUSSION THREAD

**Discussion by ochompsky** [tesla-android/issue-tracker#55] [Q&A] [answered]
If I plug the dongle into another device, internet works.  It was the same way in the first version of tesla-android.  

But after the update, if I plug it into android-os, there is no internet access.  Any thoughts on how to troubleshoot the issue?  Using Alcatel IK41UC.

**Comment by galosre** [comment, marked as answer]
If you did plug the LTE in the RPI 3 I would suggest writing again the SD card and look for any error in the process.

**Reply by ochompsky** [reply]
thanks, this did the trick
